### PR TITLE
add info about deprecation to readme

### PR DESCRIPTION
**Pull request by piotr-sepiol**

### CODE REVIEW

**Review by pmaciejp — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,4 +1,8 @@
-# Flask RAGE
+# DEPRECATED
+
+###This is no longer supported
```
Comment (niedziela) [inline README.md:3]: add space

**Review by niedziela — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (mpaliwoda) [inline README.md:3]: need a space here :D 

also maybe we'd want to add a specific date when we'll delete the repo so everybody's got the timeframe to act upon this info?

Comment (piotr-sepiol) [reply README.md]: I don't know if we want to delete this repo 🤷

Comment (mpaliwoda) [reply README.md]: ah ok, so just archiving or sth?

**Review by mpaliwoda — APPROVED** [review]

**Review by niedziela — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (piotr-sepiol) [reply README.md]: I added info about archiving

**Review by mpaliwoda — APPROVED** [review]
👍

**Comment by niedziela** [comment]
👍

**Pull request merged**